client/src/components/Profile Tabs/ProfileTabs.tsx

```diff
@@ -0,0 +1,98 @@
+// Adapted from https://material-ui.com/components/tabs/
+
+import React from 'react';
+import SwipeableViews from 'react-swipeable-views';
+import AppBar from '@material-ui/core/AppBar';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+import { useTheme } from '@material-ui/core/styles';
+import useStyles from './useStyles';
+import { ContestAPIData } from '../../interface/Contest';
+import { ContestList } from './ContestListByUsername/ContestList';
+
+interface TabPanelProps {
+  children?: React.ReactNode;
+  dir?: string;
+  index: number;
+  value: number;
+}
+
+function TabPanel(props: TabPanelProps) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`full-width-tabpanel-${index}`}
+      aria-labelledby={`full-width-tab-${index}`}
+      {...other}
+    >
+      {value === index && (
+        <Box p={3}>
+          <Typography>{children}</Typography>
```

Comment:
I believe that using Typography here inserts a <p> that causes a DOM warning:
`Warning: validateDOMNesting(...): <p> cannot appear as a descendant of <p>.`

Reply:
Thanks for reviewing, Eric! I'm seeing that error on the Notifications feature as well. Will check those too